factorial.js

```diff
@@ -0,0 +1,11 @@
+var factorial = function(num) { var total = 1;
+  if (num < 0) { return undefined }
+  var total = 1;
+  for (i = 0; i <= num; i++) {
+    if (i === 0) { continue }
```

Comment:
rather than using continue and if/else could we just check `if (i !==0) { total *= i}`